src/core/engine.js

```diff
@@ -1137,6 +1190,28 @@ function engineFactory({
             return this;
         },
 
+        /**
+         * Corrects the expression if needed.
+         * @returns {calculator}
+         * @fires expression after the expression has been changed
+         * @fires position after the position has been changed
+         * @fires replace after the expression has been replaced
+         * @fires correct after the expression has been corrected
+         */
+        correct() {
+            const tokensList = this.getTokens();
+            const correctedTokens = applyListStrategies(tokensList, correctStrategies);
+            const correctedExpression = expressionHelper.build(correctedTokens);
+
+            if (correctedExpression !== expression) {
```

Comment:
offtopic: it would be nice to think about if since you already calculate the correctedExpression if it deserves the time of selecting if apply it or not rather than just overwriting it :D